locations/storefinders/uberall.py

```diff
@@ -50,3 +52,26 @@ def parse(self, response, **kwargs):
 
     def parse_item(self, item, feature, **kwargs):
         yield item
+
+    def storefinder_exists(response: Response) -> bool | Request:
```

Comment:
I also observe some other reliable clues that Uberall is being used:

`<div id="store-finder-widget" data-key=`

`<div class="mod-uberall-store-locator" data-t-name="UberallStoreLocator"`